AzurePipelinesToGitHubActionsConverter/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelines/AzurePipelinesRoot.cs

```diff
@@ -5,42 +5,9 @@
 
 namespace AzurePipelinesToGitHubActionsConverter.Core.AzurePipelines
 {
-    //name: string  # build numbering format
-    //resources:
-    //  containers: [ containerResource ]
-    //  repositories: [ repositoryResource ]
-    //variables: { string: string } | [ variable | templateReference ]
-    //trigger: trigger
-    //pr: pr
-    //stages: [ stage | templateReference ]
-
-    public class AzurePipelinesRoot
+    public class AzurePipelinesRoot<T> : AzurePipelinesRootBase<T>
     {
-        public string name { get; set; }
-        //TODO: add resources support
```

Comment:
Look at all these TODO's - You have a lot of work to do.

Reply:
Thanks Scott, I'm going to assign this one to you. You identified the problem, you own it. #8